### PR TITLE
Add a troubleshooting guide for Sensu Go

### LINKED ISSUE #837: Guide: Troubleshooting Sensu Go
Create a new guide at /guides/troubleshooting, based on https://docs.sensu.io/sensu-core/1.6/guides/troubleshooting/

### PR DESCRIPTION
**Pull request by cwjohnston**
## Description

Adds a troubleshooting guide for Sensu Go. Content currently limited to information related to service logging.

## Motivation and Context

As an operator of Sensu Go I need some understanding of how to investigate unexpected behavior and other failures. This PR provides a starting point for collecting information related to such troubleshooting efforts.

Closes #837

## Review Instructions

see `/sensu-go/5.0/guides/troubleshooting`

### CODE REVIEW

**Review by Blue0ctober — APPROVED** [review]

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "How to troubleshoot Sensu Go services"
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
title: "Troubleshooting"
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+linkTitle: "Troubleshooting Sensu Go"
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+weight: 20
+version: "5.0"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-5.0:
+    parent: guides
+---
+
+## Service Logging
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
## Service logging
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Logs produced by Sensu services -- i.e. sensu-backend and sensu-agent -- are
+often the best place to start when troubleshooting a variety of issues. 
+
+### Log Levels
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
### Log levels
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+These log levels can be configured by specifying the desired log level as the
+value of `log-level` in the service configuration file (e.g. `agent.yml` or
+`backend.yml` configuration files), or as an argument to the `--log-level`
+command line flag:
+
+{{< highlight shell >}}
+sensu-agent start --log-level debug
+{{< /highlight >}}
+
+Changes to log level via configuration file or command line arguments requires
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
Changes to log level via configuration file or command line arguments require
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+restarting the service.
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]: Please link to where users can find how to do this

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+command line flag:
+
+{{< highlight shell >}}
+sensu-agent start --log-level debug
+{{< /highlight >}}
+
+Changes to log level via configuration file or command line arguments requires
+restarting the service. 
+
+### Log File Locations
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
### Log file locations
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+Sensu services print [structured log messages][structured] to standard output.
+In order to capture these log messages to disk or another logging facility, Sensu services
+make use of capabilities provided by the underlying operating service's service
+management. For example, logs are sent to the journald when systemd is the service manager,
+whereas log messages are redirected to `/var/log/sensu` when running under sysv
+init schemes.
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md:52]: 💯

**Review by apaskulin — APPROVED** [review]
Great work! Just a few comments then :shipit:

File: content/sensu-go/5.0/guides/troubleshooting.md (hunk 6 above)
Comment (cwjohnston) [reply content/sensu-go/5.0/guides/troubleshooting.md]: @apaskulin Do we have existing documentation on managing Sensu Go services? I’ve looked briefly and not found it.

Comment (apaskulin) [reply content/sensu-go/5.0/guides/troubleshooting.md]: https://docs.sensu.io/sensu-go/5.0/reference/agent/#restarting-the-service and https://docs.sensu.io/sensu-go/5.0/reference/backend/#restarting-the-service

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+| Debian       | <= 7       | log file     | {{< highlight shell >}}tail --follow /var/log/sensu/sensu-${service}{{< /highlight >}} |
+| Windows      | Any        | log file     | {{< highlight powershell >}}Get-Content -  Path "C:\scripts\test.txt" -Wait{{< /highlight >}}
+
+_NOTE: Platform versions described above are for reference only and do not
+supercede the documented [supported platforms][platforms]._
+
+[structured]: https://dzone.com/articles/what-is-structured-logging
+[journalctl]: https://www.digitalocean.com/community/tutorials/how-to-use-journalctl-to-view-and-manipulate-systemd-logs
+[platforms]: ../../getting-started/platforms
+[agent-ref]: ../../reference/agent/#stopping-the-service
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
[agent-ref]: ../../reference/agent/#restarting-the-service
```

File: content/sensu-go/5.0/guides/troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+[backend-ref]: ../../reference/backend/#stopping-the-service
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/troubleshooting.md]:
```suggestion
[backend-ref]: ../../reference/backend/#restarting-the-service
```

**Pull request merged**